plugins/rikki/heroeslounge/components/manageteam/default.htm

```diff
@@ -1,4 +1,4 @@
-{% if (__SELF__.user.sloth.is_captain == 1 and __SELF__.user.sloth.team_id == __SELF__.team.id) or (__SELF__.user.sloth.is_divs_captain == 1 and __SELF__.user.sloth.divs_team_id == __SELF__.team.id) %}
+{% if __SELF__.user.sloth.isCaptain(__SELF__.team) %}
```

Comment:
I looked at the implementation of this method:
https://github.com/Fabian-Sommer/HeroesLounge/blob/master/plugins/rikki/heroeslounge/models/Sloth.php

And it does not have the correct logic at all.  We should update the logic there to match the logic you replaced here, as that looks correct.

Reply:
You're correct, I assumed the logic in there was fine, but it seems that it is not. Will change to properly reflect that the user is indeed the captain of the specified team